<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminReportDialog</class>
 <widget class="QDialog" name="AdminReportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>365</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="username_lb">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>370</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>370</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>用户：</string>
   </property>
  </widget>
  <widget class="QLabel" name="type_lb">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>365</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>投诉/报修</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="deal_btn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>去处理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Processed_btn">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>430</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>已处理</string>
   </property>
  </widget>
  <widget class="QLabel" name="image_lb1">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>160</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/image/null_image.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="image_lb2">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>160</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/image/null_image.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="image_lb3">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>160</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/image/null_image.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="image_lb4">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>270</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/image/null_image.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="image_lb5">
   <property name="geometry">
    <rect>
     <x>135</x>
     <y>270</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/image/null_image.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextEdit" name="content_edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>341</width>
     <height>111</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>400</y>
     <width>54</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>位置：</string>
   </property>
  </widget>
  <widget class="QLabel" name="address_lb">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>370</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>手机号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="phone_lb">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>370</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>255</x>
     <y>270</y>
     <width>91</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/image/image/null_image.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
